src/librustc_mir/interpret/place.rs

```diff
@@ -361,12 +361,14 @@ where
     pub fn mplace_access_checked(
         &self,
         mut place: MPlaceTy<'tcx, M::PointerTag>,
+        force_align: Option<Align>,
```

Comment:
please leave a comment on this argument noting that is is only used to allow stacked borrows in miri to handle raw borrows.